src/filter-summary/index.jsx

```diff
@@ -1,31 +1,74 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
+import omit from 'lodash/omit';
+import capitalize from 'lodash/capitalize';
+import qs from 'qs';
 
 const FilterSummary = ({
   total,
   filtered,
-  filteredLabel = `Showing ${filtered} of ${total} results`,
-  allShowingLabel = `All ${total} results`
-}) => (
-  <h2 className="filter-summary">
-    {
-      filtered !== total
-        ? filteredLabel
-        : allShowingLabel
-    }
-  </h2>
-);
+  filteredLabel = filtered === 1 ? `${filtered} result` : `${filtered} results`,
+  allShowingLabel = total === 1 ? `${total} result` : `${total} results`,
```

Comment:
```
function pluralify(num) {
  return `${num} ${num === 1 ? 'result' : 'results'}`
}
```
?

Reply:
also, this seems like the wrong place to perform this logic, as it is _only_ performed if these arguments are omitted

Reply:
We can probably remove these as props as I don't think they're actually passed in anywhere?

Reply:
that sounds like a great plan